src/env.cc

```diff
@@ -33,6 +34,24 @@ using worker::Worker;
 
 #define kTraceCategoryCount 1
 
+// TODO(@jasnell): Likely useful to move this to util or node_internal to
+// allow reuse. But since we're not reusing it yet...
+class TraceEventScope {
+ public:
+  TraceEventScope(const char* category,
+                  const char* name,
+                  void* id) : category_(category), name_(name), id_(id) {
+    TRACE_EVENT_NESTABLE_ASYNC_BEGIN0(category_, name_, id);
+  }
+  ~TraceEventScope() {
+    TRACE_EVENT_NESTABLE_ASYNC_END0(category_, name_, id_);
+  }
+ private:
```

Comment:
style nit: blank line before `private:` and between the methods (the linter should complain about the former, I think?)

Reply:
doh... I had noticed that also but forgot to fix it before pushing.